book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
```

Comment:
* I suggest you make `the book itself be sent as a parameter when creating a new rental achieving the 'has-many' association` HINT 💭: you could write something like this `Rental.new(date, self, person) ` so that for every rental created it is associated with a book(self) hence the association `A book has many rentals `

Reply:
Resolved